qa/tasks/cephfs/test_failover.py

```diff
@@ -706,6 +706,66 @@ def test_rank_stopped(self):
         status = self._confirm_single_replay()
         self.assertTrue(standby_count, len(list(status.get_standbys())))
 
+    def is_there_only_1_cache_oversized_warn(self):
+        health_report = self.get_ceph_cmd_stdout('health detail')
+        log.info(health_report)
+        for line in health_report.split('\n'):
+            if line.find('MDS_CACHE_OVERSIZED') != -1:
+                self.assertIn('1 MDSs report oversized cache', line)
+                return True
+        return False
+
+    def test_health_warn_oversize_cache(self):
+        '''
+        Test that when MDS cache size crosses the limit, health warning is
+        not printed standy-replay MDS.
+
+        Tests: https://tracker.ceph.com/issues/63514
```

Comment:
Really required?

Reply:
Good to have, also doesn't hurt to have. But I am okay removing it if you don't want it.

Reply:
I don't think we follow mentioning trakers generally, unless really really required for some reason.